### DISCUSSION TITLE
Blank screen not lunching builder

### DISCUSSION THREAD

**Discussion by fgdata3** [Budibase/budibase#6172] [Help]
Hi I have installed Budibase on Ubuntu using Portainer using this template https://raw.githubusercontent.com/Budibase/budibase/master/hosting/portainer/template.json.
All containers are running except budibase_couch-init_1.
When I connect to 10,xxx,xxx,xxx:10000 all I get is a black screen and after several refreshes I will get a bad gateway 502 error.
this is the BBproxy log
2022/06/02 05:02:46 [error] 31#31: *341 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 10.xxx.xxx.35, server: _, request: "GET /api/global/configs/checklist?tenantId=default HTTP/1.1", upstream: "http://172.18.0.7:4003/api/global/configs/checklist?tenantId=default", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:02:46 +0000] "GET /api/global/configs/checklist?tenantId=default HTTP/1.1" 504 4363 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
2022/06/02 05:13:46 [notice] 1#1: signal 3 (SIGQUIT) received, shutting down
2022/06/02 05:13:46 [notice] 31#31: gracefully shutting down
2022/06/02 05:13:46 [notice] 32#32: gracefully shutting down
2022/06/02 05:13:46 [notice] 32#32: exiting
2022/06/02 05:13:46 [notice] 31#31: exiting
2022/06/02 05:13:46 [notice] 32#32: exit
2022/06/02 05:13:46 [notice] 31#31: exit
2022/06/02 05:13:46 [notice] 1#1: signal 17 (SIGCHLD) received from 31
2022/06/02 05:13:46 [notice] 1#1: worker process 31 exited with code 0
2022/06/02 05:13:46 [notice] 1#1: signal 29 (SIGIO) received
2022/06/02 05:13:46 [notice] 1#1: signal 17 (SIGCHLD) received from 32
2022/06/02 05:13:46 [notice] 1#1: worker process 32 exited with code 0
2022/06/02 05:13:46 [notice] 1#1: exit
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: IPv6 listen already enabled
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2022/06/02 05:14:24 [notice] 1#1: using the "epoll" event method
2022/06/02 05:14:24 [notice] 1#1: nginx/1.21.6
2022/06/02 05:14:24 [notice] 1#1: built by gcc 10.2.1 20210110 (Debian 10.2.1-6) 
2022/06/02 05:14:24 [notice] 1#1: OS: Linux 5.4.0-113-generic
2022/06/02 05:14:24 [notice] 1#1: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2022/06/02 05:14:24 [notice] 1#1: start worker processes
2022/06/02 05:14:24 [notice] 1#1: start worker process 24
2022/06/02 05:14:24 [notice] 1#1: start worker process 25
2022/06/02 05:19:15 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.xxx.xxx.35, server: _, request: "GET /builder HTTP/1.1", upstream: "http://172.18.0.8:4002/builder", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:15 +0000] "GET /builder HTTP/1.1" 502 4363 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
2022/06/02 05:19:16 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.xxx.xxx.35, server: _, request: "GET /builder HTTP/1.1", upstream: "http://172.18.0.8:4002/builder", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:16 +0000] "GET /builder HTTP/1.1" 502 4363 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:16 +0000] "GET /favicon.ico HTTP/1.1" 403 273 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
2022/06/02 05:19:17 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.xxx.xxx.35, server: _, request: "GET /builder HTTP/1.1", upstream: "http://172.18.0.8:4002/builder", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:17 +0000] "GET /builder HTTP/1.1" 502 4363 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
2022/06/02 05:19:17 [error] 24#24: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.xxx.xxx.35, server: _, request: "GET /builder HTTP/1.1", upstream: "http://172.18.0.8:4002/builder", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:17 +0000] "GET /builder HTTP/1.1" 502 4363 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:17 +0000] "GET /favicon.ico HTTP/1.1" 403 273 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:18 +0000] "GET /builder HTTP/1.1" 200 465 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /builder/assets/vendor.5e1fe644.css HTTP/1.1" 200 3520 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /builder/assets/index.8a6f249b.css HTTP/1.1" 200 86895 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /builder/assets/index.c36e45db.js HTTP/1.1" 200 557610 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /builder/assets/vendor.9bf2fcb4.js HTTP/1.1" 200 683474 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /builder/assets/favicon.e7fc7733.png HTTP/1.1" 200 6835 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:19 +0000] "GET /api/global/self HTTP/1.1" 403 80 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /builder HTTP/1.1" 200 465 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
2022/06/02 05:19:23 [info] 25#25: *4 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 10.xxx.xxx.35, server: _, request: "GET /api/global/configs/checklist?tenantId=default HTTP/1.1", upstream: "http://172.18.0.6:4003/api/global/configs/checklist?tenantId=default", host: "10.xxx.xxx.80:10000", referrer: "http://10.xxx.xxx.80:10000/builder"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /api/global/configs/checklist?tenantId=default HTTP/1.1" 499 0 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /builder/assets/vendor.5e1fe644.css HTTP/1.1" 200 3520 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /builder/assets/index.8a6f249b.css HTTP/1.1" 200 86895 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /builder/assets/index.c36e45db.js HTTP/1.1" 200 557610 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:23 +0000] "GET /builder/assets/vendor.9bf2fcb4.js HTTP/1.1" 200 683474 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:24 +0000] "GET /builder/assets/favicon.e7fc7733.png HTTP/1.1" 200 6835 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:24 +0000] "GET /api/global/self HTTP/1.1" 403 80 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"
10.xxx.xxx.35 - - [02/Jun/2022:05:19:25 +0000] "GET /builder HTTP/1.1" 200 465 "http://10.xxx.xxx.80:10000/builder" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53"

BBworker log
    req: {
      "id": 6,
      "method": "GET",
      "url": "/api/global/configs/checklist?tenantId=default",
      "headers": {
        "host": "10.xxx.xxx.80",
        "connection": "close",
        "accept": "application/json",
        "x-budibase-app-id": "",
        "x-budibase-api-version": "1",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53",
        "referer": "http://10.xxx.xxx.80:10000/builder",
        "accept-encoding": "gzip, deflate",
        "accept-language": "en-US,en;q=0.9,en-AU;q=0.8",
        "cookie": "_pk_id.1.f2af=093da363a16361d2.1654071946.; ajs_anonymous_id=a3921b81-6edb-453a-a931-0457c57646b4; mp_70b8ea94d623dd857fb555a76d11f944_mixpanel=%7B%22distinct_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22%24device_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22mp_lib%22%3A%20%22Segment%3A%20web%22%2C%22%24initial_referrer%22%3A%20%22%24direct%22%2C%22%24initial_referring_domain%22%3A%20%22%24direct%22%7D; SESSION=e340ab3e-ad02-4461-b01a-e8d7ae43ae88"
      },
      "remoteAddress": "::ffff:172.18.0.7",
      "remotePort": 52760
    }
ERROR [1654147565330] (29 on fdc736c936b9):
    req: {
      "id": 7,
      "method": "GET",
      "url": "/api/global/self",
      "headers": {
        "host": "10.xxx.xxx.80",
        "connection": "close",
        "accept": "application/json",
        "x-budibase-app-id": "",
        "x-budibase-api-version": "1",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53",
        "referer": "http://10.xxx.xxx.80:10000/builder",
        "accept-encoding": "gzip, deflate",
        "accept-language": "en-US,en;q=0.9,en-AU;q=0.8",
        "cookie": "_pk_id.1.f2af=093da363a16361d2.1654071946.; ajs_anonymous_id=a3921b81-6edb-453a-a931-0457c57646b4; mp_70b8ea94d623dd857fb555a76d11f944_mixpanel=%7B%22distinct_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22%24device_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22mp_lib%22%3A%20%22Segment%3A%20web%22%2C%22%24initial_referrer%22%3A%20%22%24direct%22%2C%22%24initial_referring_domain%22%3A%20%22%24direct%22%7D; SESSION=832657a4-3ab5-403b-9e69-70740e353d8c"
      },
      "remoteAddress": "::ffff:172.18.0.7",
      "remotePort": 53168
    }
    err: {
      "type": "ForbiddenError",
      "message": "Unauthorized - no public worker access",
      "stack":
          ForbiddenError: Unauthorized - no public worker access
              at Object.throw (/app/node_modules/koa/lib/context.js:97:11)
              at /app/dist/api/index.js:109:18
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at /app/node_modules/@koa/router/lib/router.js:368:16
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at /app/node_modules/@budibase/pro/dist/index.js:1:316892
              at Generator.next (<anonymous>)
              at /app/node_modules/@budibase/pro/dist/index.js:1:3817
              at new Promise (<anonymous>)
              at j (/app/node_modules/@budibase/pro/dist/index.js:1:3562)
      "status": 403,
      "statusCode": 403,
      "expose": true
    }
Error fetching before cache -  FetchError: request to http://0.0.0.1/_all_dbs failed, reason: connect ETIMEDOUT 0.0.0.1:80
    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (events.js:412:35)
    at ClientRequest.emit (domain.js:475:12)
    at Socket.socketErrorListener (_http_client.js:475:9)
    at Socket.emit (events.js:400:28)
    at Socket.emit (domain.js:475:12)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
ERROR [1654147696435] (29 on fdc736c936b9): request to http://0.0.0.1/_all_dbs failed, reason: connect ETIMEDOUT 0.0.0.1:80
    message: "request to http://0.0.0.1/_all_dbs failed, reason: connect ETIMEDOUT 0.0.0.1:80"
    type: "system"
    errno: "ETIMEDOUT"
    code: "ETIMEDOUT"
    expose: false
    statusCode: 500
    status: 500
    stack: "FetchError: request to http://0.0.0.1/_all_dbs failed, reason: connect ETIMEDOUT 0.0.0.1:80\n    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1491:11)\n    at ClientRequest.emit (events.js:412:35)\n    at ClientRequest.emit (domain.js:475:12)\n    at Socket.socketErrorListener (_http_client.js:475:9)\n    at Socket.emit (events.js:400:28)\n    at Socket.emit (domain.js:475:12)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)"
    req: {
      "id": 8,
      "method": "GET",
      "url": "/api/global/configs/checklist?tenantId=default",
      "headers": {
        "host": "10.xxx.xxx.80",
        "connection": "close",
        "accept": "application/json",
        "x-budibase-app-id": "",
        "x-budibase-api-version": "1",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.53",
        "referer": "http://10.xxx.xxx.80:10000/builder",
        "accept-encoding": "gzip, deflate",
        "accept-language": "en-US,en;q=0.9,en-AU;q=0.8",
        "cookie": "_pk_id.1.f2af=093da363a16361d2.1654071946.; ajs_anonymous_id=a3921b81-6edb-453a-a931-0457c57646b4; mp_70b8ea94d623dd857fb555a76d11f944_mixpanel=%7B%22distinct_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22%24device_id%22%3A%20%2218122106745185-079a41f60db3fe-4c647e53-1fa400-18122106746af8%22%2C%22mp_lib%22%3A%20%22Segment%3A%20web%22%2C%22%24initial_referrer%22%3A%20%22%24direct%22%2C%22%24initial_referring_domain%22%3A%20%22%24direct%22%7D; SESSION=832657a4-3ab5-403b-9e69-70740e353d8c"
      },
      "remoteAddress": "::ffff:172.18.0.7",
      "remotePort": 53170
    }

BBapps log
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)
    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)
    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)
    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22
    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)
bb-alert: Error performing migrations. Exiting. : {"stack":"TypeError: Cannot read property 'user_email_view_casing' of undefined\n    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)\n    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)\n    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)\n    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22\n    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)","message":"Cannot read property 'user_email_view_casing' of undefined"}
Server Closed
Done in 134.05s.
yarn run v1.22.19
$ node dist/index.js
Running migrations
Budibase running on {"address":"::","family":"IPv6","port":4002}
FetchError: request to http://0.0.0.1/global-db/ failed, reason: connect ETIMEDOUT 0.0.0.1:80
    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (events.js:412:35)
    at ClientRequest.emit (domain.js:475:12)
    at Socket.socketErrorListener (_http_client.js:475:9)
    at Socket.emit (events.js:400:28)
    at Socket.emit (domain.js:475:12)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
[Tenant: default] [Migration: user_email_view_casing] [DB: global-db] Error:  TypeError: Cannot read property 'user_email_view_casing' of undefined
    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)
    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)
    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)
    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22
    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)
bb-alert: Error performing migrations. Exiting. : {"stack":"TypeError: Cannot read property 'user_email_view_casing' of undefined\n    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)\n    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)\n    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)\n    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22\n    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)","message":"Cannot read property 'user_email_view_casing' of undefined"}
Server Closed
Done in 132.81s.
yarn run v1.22.19
$ node dist/index.js
Running migrations
Budibase running on {"address":"::","family":"IPv6","port":4002}
FetchError: request to http://0.0.0.1/global-db/ failed, reason: connect ETIMEDOUT 0.0.0.1:80
    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (events.js:412:35)
    at ClientRequest.emit (domain.js:475:12)
    at Socket.socketErrorListener (_http_client.js:475:9)
    at Socket.emit (events.js:400:28)
    at Socket.emit (domain.js:475:12)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
[Tenant: default] [Migration: user_email_view_casing] [DB: global-db] Error:  TypeError: Cannot read property 'user_email_view_casing' of undefined
    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)
    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)
    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)
    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22
    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)
bb-alert: Error performing migrations. Exiting. : {"stack":"TypeError: Cannot read property 'user_email_view_casing' of undefined\n    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)\n    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)\n    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)\n    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22\n    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)","message":"Cannot read property 'user_email_view_casing' of undefined"}
Server Closed
Done in 134.00s.
yarn run v1.22.19
$ node dist/index.js
Running migrations
Budibase running on {"address":"::","family":"IPv6","port":4002}
FetchError: request to http://0.0.0.1/global-db/ failed, reason: connect ETIMEDOUT 0.0.0.1:80
    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1491:11)
    at ClientRequest.emit (events.js:412:35)
    at ClientRequest.emit (domain.js:475:12)
    at Socket.socketErrorListener (_http_client.js:475:9)
    at Socket.emit (events.js:400:28)
    at Socket.emit (domain.js:475:12)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
[Tenant: default] [Migration: user_email_view_casing] [DB: global-db] Error:  TypeError: Cannot read property 'user_email_view_casing' of undefined
    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)
    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)
    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)
    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22
    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)
bb-alert: Error performing migrations. Exiting. : {"stack":"TypeError: Cannot read property 'user_email_view_casing' of undefined\n    at /app/node_modules/@budibase/backend-core/src/migrations/index.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async exports.doWithDB (/app/node_modules/@budibase/backend-core/src/db/index.js:62:12)\n    at async runMigration (/app/node_modules/@budibase/backend-core/src/migrations/index.js:50:5)\n    at async internal (/app/node_modules/@budibase/backend-core/src/context/index.js:90:14)\n    at async /app/node_modules/@budibase/backend-core/src/context/FunctionContext.js:23:22\n    at async exports.runMigrations (/app/node_modules/@budibase/backend-core/src/migrations/index.js:113:7)","message":"Cannot read property 'user_email_view_casing' of undefined"}
Server Closed
Done in 132.91s.
yarn run v1.22.19
$ node dist/index.js
Running migrations
Budibase running on {"address":"::","family":"IPv6","port":4002}

**Comment by fgdata3** [comment]
Hi I cannot get Budibase to load can anyone please help

**Comment by melohagan** [comment]
Hi @fgdata3 

Maybe some of the images are out of date. You could try stopping all of the containers and running a: `docker image prune -a`

Then deploy the template to pull down the latest images.

**Reply by fgdata3** [reply]
Thanks. I did update all of the containers still same issue we gave up and are now running it on VMware Photon OS with portainer
